pygmt/clib/session.py

```diff
@@ -338,19 +338,12 @@ def create(self, name: str):
         name
             A name for this session. Doesn't really affect the outcome.
         """
```

Comment:
If `self.session_pointer` is defined and is not None, then we know we have an open session. The `try..except` statement was added in 2018 (#210) and is not necessary.

Reply:
It's impossible that the `msg` assignment and the `raise GMTCLibError` lines can raise a `GMTCLibNoSessionError` exception, so we should be safe.

Reply:
Reverted back to `try..except` in da9c5e17abdffbba49fc49bb9d82f49b2a794bc2 and improve the comments to make it easier to understand.